src/App.scss

```diff
@@ -87,7 +87,7 @@ section.body {
 
 
   > nav.menu {
-    width: 100%;
+    width: auto;
```

Comment:
is it related to cookies implementation? or from other branch

Reply:
I have fixed nav menu